css/user.css

```diff
@@ -0,0 +1,58 @@
+.user-body {
+    display: flex;
+    justify-content: center;
```

Comment:
Can we make a class for "flex properties" and use it across the file? As, it's been used more than once.

Reply:
This is a good suggestion. Will keep it in mind moving forward. Thanks!